tests/api/test_api.py

```diff
@@ -12,6 +12,7 @@
 
 SERVER_URL = "http://127.0.0.1:%s" % (tests.api.server.PORT)
 FORMAT_STR = "\n--- Expected ---\n%s\n--- Actual ---\n%s\n---\n"
+PYTHON_ERROR_PREFIX = "Failed to complete operation: "
```

Comment:
All errors have the same prefix because they are all being processed by the following code in [tests/api/common.py](https://github.com/edulinq/autograder-py/blob/6ffb1f897d17bb1566615443bd875af0d078e1d9/autograder/api/common.py#L84).

When the tests/api/server.py responds with the error message, it needs to get the raw error message from the content that is given to it by Handler._next_response_queue.

I don't think it makes sense to include another field in the JSON when it can be easily built from the server error message. I chose to standardize the prefix so it's more clear.

Thoughts?

Reply:
The errors are generated in the same place now.
But there is nothing saying it will be that way in the future.
This change would paint us in a small corner and force all messages to have the same prefix.

For example, it would make total sense to handle 401 specially.